frontend/public/components/sidebars/resource-sidebar.jsx

```diff
@@ -1,5 +1,6 @@
 import * as React from 'react';
 import { CloseIcon, DownloadIcon, InfoCircleIcon, PasteIcon } from '@patternfly/react-icons';
+import { Button } from '@patternfly/react-core';
```

Comment:
This file is changing significantly in #2889, so I'd hold off on migrating these buttons.

Reply:
Now included in #2889, so please drop the changes in this PR.